gems/pending/VolumeManager/VolMgrPlatformSupportLinux.rb

```diff
@@ -11,10 +11,9 @@ def init
         @vimVm = nil
         
         if !$miqHostCfg
-            $log.warn "VolMgrPlatformSupportLinux: $miqHostCfg not set"
-            return
+          return
```

Comment:
now that this is simplified, should this be one-lined to:
`return if !$miqHostCfg` or `return unless $miqHostCfg`